src/main/scala/cromwell/engine/backend/jes/JesBackend.scala

```diff
@@ -568,7 +569,7 @@ case class JesBackend(actorSystem: ActorSystem)
       status match {
         case Run.Success(events) if backendCall.runtimeAttributes.failOnStderr && stderrLength.intValue > 0 =>
           // returnCode will be None if it couldn't be downloaded/parsed, which will yield a null in the DB
-          FailedExecutionHandle(new Throwable(s"${log.tag} execution failed: stderr has length $stderrLength"), returnCode.toOption).future
+          FailedExecutionHandle(new Throwable(s"${log.tag} execution failed: stderr has length $stderrLength"), returnCode.toOption map ScriptReturnCode).future
```

Comment:
Hmm that would probably be an implicit class around `Try[Int]` since returnCode is a `Try`, which feels a bit hacky to me but why not. Another solution is a
```scala
implicit def toScriptReturnCode(rc: Try[Int]): Option[ScriptReturnCode] = rc.toOption map ScriptReturnCode
```
And then we can just use`returnCode` in the `FailedExecutionHandle`